JS/main_biz_owner.js

```diff
@@ -0,0 +1,91 @@
+import { initializeBizMap } from "./biz-map.js";
+import { initializeList } from "./biz-list_biz_owner.js";
+import { initFilters } from"./filter.js";
+import { initializeSearch } from "./search.js";
+
+const bizpointsResp = await fetch('data/union_sq.json');
+const bizpoints = await bizpointsResp.json();
+const map = initializeBizMap(bizpoints);
+
+initializeList(bizpoints);
+initFilters(bizpoints, map);
+initializeSearch(bizpoints);
+
+//dropdown
+const dropdown = L.control({ position: 'topright' });
+
+    dropdown.onAdd = function (map) {
+      const div = L.DomUtil.create('div', 'dropdown');
+  
+      div.innerHTML = '<button class="btn" id="filterButton">Filter</button>';
+      div.querySelector('#filterButton').addEventListener('click', toggleDropdown);
+      return div;
+    };
+  
+
+
+function toggleDropdown() {
+    const dropdownContent = document.getElementById("filterOptions");
+    dropdownContent.style.display = dropdownContent.style.display === "block" ? "none" : "block";
+  }
+
+  dropdown.addTo(map);
+
+// popup submission form
+
+function closeForm() {
+      document.getElementById("popup-form").style.display = "none";
+  }
+
+// Add this function to handle form submission
+function submitForm() {
+  // Get form data

```

Comment:
Second, you want to call `preventDefault` from within the `submitForm` function. `preventDefault` is a function on an event object that just tells the browser to _not_ do whatever it would normally do in response to the particular event. For example:
- by default, when you click a link, the browser will open the URL specified in the `href`
- by default, when you submit a form (or click a submit button within a form), the browser will send the form data to whatever URL is specified in the form's `action` attribute and reload the page

By calling `evt.preventDefault()`, the page won't reload when you click the button.

```suggestion
function submitForm(evt) {
  evt.preventDefault();
  
  // Get form data
```

After those changes, I _do_ see form data getting saved to the local storage. E.g.:
![image](https://github.com/musa-6110-fall-2023/dashboard-project/assets/146749/96d04b9b-4418-4e0c-bacd-c3673c4071f3)